modules/tinymce/src/core/test/ts/browser/keyboard/ArrowKeysCetContentEndpointNavigation.ts

```diff
@@ -0,0 +1,117 @@
+import { Keys } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyContentActions, TinyHooks, TinySelections } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import Env from 'tinymce/core/api/Env';
+
+describe('browser.tinymce.core.keyboard.ArrowKeysCetContentEndpointNavigation', () => {
+  if (!Env.browser.isFirefox()) {
+    return;
+  }
+
```

Comment:
A better solution, in this case, would be:

```suggestion
describe('browser.tinymce.core.keyboard.ArrowKeysCetContentEndpointNavigation', function() {
  if (!Env.browser.isFirefox()) {
    this.skip();
  }

```

Reply:
Andy suggested it's better to avoid skip, see:
https://github.com/tinymce/tinymce/pull/10484#discussion_r2221144146